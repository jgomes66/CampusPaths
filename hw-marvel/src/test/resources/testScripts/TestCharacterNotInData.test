# Simple test to see what happens when a character asked for
# is not in the given data

LoadGraph g oneSuperhero.csv

ListNodes g

ListChildren g A

FindPath g A B
FindPath g B C
