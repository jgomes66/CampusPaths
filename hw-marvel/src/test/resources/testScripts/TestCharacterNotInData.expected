# Simple test to see what happens when a character asked for
# is not in the given data

loaded graph g

g contains: A

the children of A in g are:

unknown: B
unknown: B
unknown: C